tutorials/io/30_reading_fnirs_data.py

```diff
@@ -46,7 +46,26 @@
 
 .. note:: The SNIRF format has provisions for many different types of fNIRS
           recordings. MNE-Python currently only supports reading continuous
-          wave data stored in the .snirf format.
+          wave or haemoglobin data stored in the .snirf format.
+
+
+Specifying the coordinate system
+--------------------------------
+
+There are a variety of coordinate systems used to specify the location of
+sensors (see :ref:`tut-source-alignment` for details). Where possible the
+coordinate system will be determined automatically when reading a SNIRF file.
+However, sometimes this is not possible and you must manually specify the
+coordinate frame the optodes are in. This is done using the ``optode_frame``
+argument when loading data.
+
+=======  ==================  =================
+Vendor   Model               ``optode_frame``
+=======  ==================  =================
+NIRx     ICBM-152 MNI        mri
+Kernel   ICBM 2009b          mri
```

Comment:
I have just taken the model names exactly as described by the vendors. I guess these might actually be the same thing?

Reply:
I don't know the bottom line but I would be very concerned for my sanity if those two were meaningfully different MNI spaces.

Reply:
haha agreed, I am pretty sure they are practically the same. Maybe I will just change them both to ICBM

Reply:
they may be slightly different: there are several versions of the ICBM-152 MNI atlas (2009a, 2009b, 2009c; and within each, different flavors, linear vs. nonlinear, symmetric vs. asymmetric). 

The 2009b is the highest resolution atlas (0.5mm isotropic) and thus the one we chose to run tissue segmentation and make meshes.

Calling them both `ICBM-152 MNI` is likely enough information.

Reply:
Yes there are different template images, but the *space* = the co-ordinate frame, ie the target of the voxel-to-world transform in the header's affine matrix, should I think be same. 

( Whether MRI template images in standard space used for registration should be called 'atlases' is debatable, but it's true that is a common terminology )

Reply:
agreed on both points.

Reply:
Thank you both for the information. I appreciate you sharing your knowledge here, I am learning lots as we go along here :)